src/operations/find_and_modify.ts

```diff
@@ -127,6 +133,11 @@ class FindAndModifyOperation extends CommandOperation<Document> {
       upsert: false
     };
 
+    // Default to returning the raw result.
+    if (!('includeResultMetadata' in options)) {
+      options.includeResultMetadata = true;
+    }
+
```

Comment:
```suggestion
    options.includeResultMetadata ??= true;
```

Reply:
```ts
this.options = { ...options };
```

In either approach, should we be avoiding modifying the input options?

Reply:
The options are already copied in `resolveOptions`

Reply:
Updated per suggestion.